Resources/public/Admin.js

```diff
@@ -103,6 +103,31 @@ var Admin = {
                 checkboxClass: 'icheckbox_square-blue',
                 radioClass: 'iradio_square-blue'
             });
+
+            var previousIndex;
+
+            jQuery('tbody .iCheck-helper', subject).click(function (evt) {
```

Comment:
Also, it's unclear to me what this function is being bound to. It does not seem to be the checkbox itself.